### PR TITLE
allow pytest to retry getting model artifacts + disable autotuning for pytorch benchmarks

### PR DESCRIPTION
**Pull request by monorimet**

### CODE REVIEW

**Comment by monorimet** [comment]
FYI @mariecwhite to unblock SHARK/IREE benchmarks on GPU + revive our CI while I work on the benchmark framework

File: shark/shark_downloader.py [hunk 1]
```diff
@@ -204,17 +204,17 @@ def download_model(
...
-        tank_dir = WORKDIR
-        gen_shark_files(model_name, frontend, tank_dir, import_args)
-        with open(filename, mode="rb") as f:
-            mlir_file = f.read()
+        print(
+            "The model data was not found. Trying to generate artifacts locally."
+        )
+        gen_shark_files(model_name, frontend, WORKDIR, import_args)
 
+    assert os.path.exists(filename)
```
Comment (dan-garvey) [inline shark/shark_downloader.py:215]: should provide error message if fail on assert due to filename still not existing after gen

File: tank/test_models.py [hunk 2]
```diff
@@ -169,18 +169,27 @@ def create_and_check_module(self, dynamic, device):
...
-                tank_url=self.tank_url,
-                frontend=self.config["framework"],
-                import_args=import_config,
-            )
-        except NoImportException:
-            pytest.xfail(
-                reason=f"Artifacts for this model/config must be generated locally. Please make sure {self.config['framework']} is installed."
-            )
-
+        dl_gen_attempts = 3
```
Comment (dan-garvey) [inline tank/test_models.py:172]: If we need this to be greater than 2 I feel like we have bigger problems. Are we hitting spurious failures often enough that a single retry wouldn't cover them?

File: shark/shark_benchmark_runner.py [hunk 3]
```diff
@@ -132,12 +132,12 @@ def benchmark_torch(self, modelname):
         frontend_model.to(torch_device)
         input.to(torch_device)
 
-        try:
-            frontend_model = torch.compile(
-                frontend_model, mode="max-autotune", backend="inductor"
-            )
-        except RuntimeError:
-            frontend_model = HFmodel.model
+        # try:
```
Comment (dan-garvey) [inline shark/shark_benchmark_runner.py:135]: dead code?

**Review by dan-garvey — CHANGES_REQUESTED** [review]

File: shark/shark_benchmark_runner.py (hunk 3 above)
Comment (monorimet) [reply shark/shark_benchmark_runner.py:135]: Will need to re-enable as soon as CUDA context issues from running inductor in a forked subprocess are resolved.

File: tank/test_models.py (hunk 2 above)
Comment (monorimet) [reply tank/test_models.py:172]: It doesn't need to be greater than 2. I'll reduce it

**Review by dan-garvey — DISMISSED** [review]

File: shark/shark_benchmark_runner.py (hunk 3 above)
Comment (dan-garvey) [reply shark/shark_benchmark_runner.py:135]: Ideally add a comment to that effect

**Review by dan-garvey — APPROVED** [review]

**Pull request merged**